users/README.md

```diff
@@ -34,6 +34,11 @@
 		"super_user": "boolean",
 		"in_discord": "boolean"
 	},
+    "archivedDetails": {
+    "updated_at": "number",
```

Comment:
```suggestion
    "updated_at": "timestamp",
```

Reply:
Unix is a number based timestamp, so keeping the type of `updated_at` as number

Reply:
Console.log(typeof Math.floor(Date.now()* 1000)) returns number